x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -923,15 +927,29 @@ private static PEMEncryptor getEncrypter(char[] password) {
         return new JcePEMEncryptorBuilder("DES-EDE3-CBC").setProvider(BC_PROV).build(password);
     }
 
-    private static <T, E extends Exception> T withPassword(String description, char[] password, Terminal terminal,
+    private static <T, E extends Exception> T withPassword(String description, boolean long_size, char[] password, Terminal terminal,
```

Comment:
Rather than `long_size`, it would be better to name this parameter something like `forNewKey` so it describes the logical purpose of the parameter.